### DISCUSSION TITLE
Node_0 added on top of the Hierarchy

### DISCUSSION THREAD

**Discussion by asanint** [iimachines/Maya2glTF#176] [Q&A]
I noticed sometimes the exported model has "Node_0" group at the very top of the hierarchy, is it possible to remove this behavior and keep the object's hierarchy intact?

**Comment by ziriax** [comment]
The extra root node is added when you export with a global scaling factor, see red marking in picture. This root node applies the scaling,  and all child nodes and meshes use the same values as in Maya. I should give that a node a good name though.

One solution is to bake the scaling factor in the vertices, like this no extra node will be created, see the green box in picture.

Another option would be to bake the scaling factor in all existing root nodes itself, but that feature doesn't exist yet; it would require adjusting animation curves etc...

![image](https://user-images.githubusercontent.com/2389359/153245876-7d22b3ef-89a8-4276-a46f-5092658e1e0b.png)
